packages/editor/src/components/parts/common/customfield/CustomFieldTable.tsx

```diff
@@ -32,12 +34,43 @@ const EMPTY_WFCUSTOMFIELD: WfCustomField = { name: '', type: 'STRING', value: ''
 const CustomFieldTable = ({ data, onChange, type }: CustomFieldTableProps) => {
   const items = useMemo<SelectItem[]>(() => Object.entries(CUSTOM_FIELD_TYPE).map(([value, label]) => ({ label, value })), []);
 
+  const { context } = useEditorContext();
+  const path = usePath();
+
+  const toWfFieldType: (type: string) => WfFieldType = typeAsString => {
+    if (typeAsString === 'TEXT') {
+      return 'TEXT';
+    }
+    if (typeAsString === 'NUMBER') {
+      return 'NUMBER';
+    }
+    if (typeAsString === 'TIMESTAMP') {
+      return 'TIMESTAMP';
+    }
+    return 'STRING';
+  };
+
+  const predefinedCustomField: WfCustomField[] = useMeta(
+    'meta/workflow/customFields',
+    { context, type: toWorkflowType(path) },
+    []
+  ).data.map<WfCustomField>(pcf => ({
+    name: pcf.name,
+    type: toWfFieldType(pcf.type),
```

Comment:
what does pcf mean?
why do we have to convert this type?

Reply:
No longer needed. I removed this mapping 👍